aws/task-definition/prod/task-definition.json

```diff
@@ -0,0 +1,249 @@
+            "value": "false"
+          },
+          {
+            "name": "vkmUrl",
+            "value": "https://api.vaylapilvi.fi"
+          },
+          {
+            "name": "tierekisteriRestApiEndPoint",
+            "value": "https://testioag.vayla.fi/trrest/devtest/"
+          },
+          {
+            "name": "tierekisteri.enabled",
+            "value": "true"
+          },
+          {
+            "name": "vallu.server.sending_enabled",
+            "value": "true"
+          },
+          {
+            "name": "vallu.server.address",
+            "value": "https://testioag.ely-keskus.fi/koontikartta/rest/stop/xml"
+          },
+          {
+            "name": "viite.importTimeStamp",
+            "value": "1510876800000"
+          },
+          {
+            "name": "http.proxySet",
+            "value": "true"
+          },
+          {
+            "name": "http.proxyHost",
+            "value": "127.0.0.1"
+          },
+          {
+            "name": "http.proxyPort",
+            "value": "3128"
+          },
+          {
+            "name": "http.nonProxyHosts",
+            "value": "172.17.*|localhost|127.*"
+          },
+          {
+            "name": "feedbackAssetsEndPoint",
+            "value": "https://digiroaddev.testivaylapilvi.fi/"
+          },
+          {
+            "name": "bonecp.jdbcUrl",
+            "value": "jdbc:postgresql://ddw6gldo8fiqt4.c8sq5c8rj3gu.eu-west-1.rds.amazonaws.com:5432/digiroad2"
+          },
+          {
+            "name": "bonecp.username",
+            "value": "digiroad2dbuser"
+          },
+          {
+            "name": "tierekisteri.username",
+            "value": "trrestoth"
+          },
+          {
+            "name": "tierekisteri.old.username",
+            "value": "trrest"
+          },
+          {
+            "name": "oag.username",
+            "value": "svc_clouddr"
+          },
+          {
+            "name": "emailTo",
+            "value": "info@digiroad.fi"
+          },
+          {
+            "name": "emailHost",
+            "value": "email-smtp.eu-west-1.amazonaws.com"
+          },
+          {
+            "name": "emailPort",
+            "value": "587"
+          },
+          {
+            "name": "emailFrom",
+            "value": "noreply.digiroad@vaylapilvi.fi"
+          },
+          {
+            "name": "env",
+            "value": "Dev"
+          },
+          {
+            "name": "caching",
+            "value": "true"
+          },
+          {
+            "name": "cacheHostname",
+            "value": "roadlinkcache.4xakoi.cfg.euw1.cache.amazonaws.com"
+          },
+          {
+            "name": "cacheHostPort",
+            "value": "11211"
+          },
+          {
+            "name": "cacheTTL",
+            "value": "72000"
+          },
+          {
+            "name": "rasterServiceUrl",
+            "value": "https://testioag.vayla.fi/rasteripalvelu-mml"
```

Comment:
varmaan voi olla näin, myöhemmin pitää muuttaa 
https://api.vaylapilvi.fi/rasteripalvelu-mml/¨
sitten kun ollaan lisätty api avain